frontend/src/metabase/query_builder/components/notebook/steps/JoinStep/JoinTablePicker/JoinTablePicker.styled.tsx

```diff
@@ -1,16 +1,35 @@
 import styled from "@emotion/styled";
+import { css } from "@emotion/react";
 import IconButtonWrapper from "metabase/components/IconButtonWrapper/IconButtonWrapper";
 import { color } from "metabase/lib/colors";
-import { NotebookCell } from "../../../NotebookCell";
+import { NotebookCell, NotebookCellItem } from "../../../NotebookCell";
 
 export const PickerButton = styled.button`
   color: inherit;
   font-weight: inherit;
-  cursor: pointer;
+  cursor: ${props => (props.disabled ? "auto" : "pointer")};
 `;
 
 export const ColumnPickerButton = styled(IconButtonWrapper)`
   padding: ${NotebookCell.CONTAINER_PADDING};
   opacity: 0.5;
   color: ${color("white")};
 `;
+
+export const PickerNotebookCellItem = styled(NotebookCellItem)<{
+  canChangeTable?: boolean;
+}>`
+  ${NotebookCellItem.Content} {
```

Comment:
Could you pass props instead of overriding styles?

Reply:
I wasn't sure about it because `NotebookCellItem` is a bit of a mess already, but after adding another prop I think it's actually better now. Thanks!